holidays/countries/gambia.py

```
@@ -0,0 +1,93 @@
+            self[date(year, 1, 29)] = "Prophet's Birthday (Mawlid al-Nabi)"  # Approximate
+        elif year == 2026:
+            self[date(year, 1, 19)] = "Prophet's Birthday (Mawlid al-Nabi)"  # Approximate
+        # Add more years as needed
+
+        # Christmas Day
+        self[date(year, 12, 25)] = "Christmas Day"
+
+
+if __name__ == "__main__":
+    gm_holidays = GM(years=2025)
+    print(gm_holidays)
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)